Tasks/PerformActionOnMachineGroup/task.json

```diff
@@ -22,7 +22,7 @@
             "type": "string", 
             "label": "Machine Group",
             "required": true,
-            "helpMarkDown": "Name of the Machine Group on which the operation is to be performed."
+            "helpMarkDown": "Name of the target Machine Group for performing the action."
```

Comment:
maintain consistent casing of Machine Group in the help mark down. Follow sentence case